### PR TITLE
Fix output of alpha channel floats

### LINKED ISSUE #860: [SvgDrawVisitor] - Problem with Locale when String.format(...) is used
Dear @johnmay,

I noticed a weird problem when trying to use a transparent background color and writing the `Depiction` to SVG format.

The problem is that you assume that the Locale used by String.format() will be en/US of any other locale (ROOT) that uses the dot (.) as a decimal separator, however in my Locale (pt/PT) the default decimal separator is the comma (,). This causes the method `toStr(Color col)`:

```java
String toStr(Color col) {
        if (col.getAlpha() == 255) {
            return String.format("#%06X", (0xFFFFFF & col.getRGB()));
        } else {
            return String.format("rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
        }
    }
```
to write something like `rgba(255,0,0,0,00)` to the SVG tree instead of `rgba(255,0,0,0.00)`  when the alpha channel in the `col` is different than 255 (0.00 for alpha value of 0).

What happens as a result is that the element will be colored weirdly (in my case was always coloring red instead of being transparent) I noticed this in the background rectangle, but this will happen for other SVG filled elements.

The fix is quite easy, simply use `LOCALE.ROOT` as an argument to the `String.format(...)` method:

```java
return String.format(Locale.ROOT, "rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
```

thanks for the great library

### PR DESCRIPTION
**Pull request by egonw**
Fixed #860

### CODE REVIEW

File: app/depict/src/main/java/org/openscience/cdk/depict/SvgDrawVisitor.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ String toStr(Color col) {
         if (col.getAlpha() == 255) {
             return String.format("#%06X", (0xFFFFFF & col.getRGB()));
         } else {
-            return String.format("rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
+            return String.format(Locale.US, "rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
```
Comment (johnmay) [inline app/depict/src/main/java/org/openscience/cdk/depict/SvgDrawVisitor.java:201]: I think ``Locale.ROOT`` is the normal way to do this.

**Locale.ROOT states that it is the language/country neutral locale for the locale sensitive operations**

I think it's more if someone comes along and tries to internationalise the code they put the commas back in ;-).

Comment (egonw) [reply app/depict/src/main/java/org/openscience/cdk/depict/SvgDrawVisitor.java:201]: I'm on it.

**Comment by johnmay** [comment]
Shall we just put the sonar cloud key in the GH action?

**Comment by johnmay** [comment]
Well that's what we have, the issue is even you can't get it to run with a PR from a branch in your fork

**Comment by johnmay** [comment]
I think there is a way to do it with GITHUB_TOKEN instead of SONAR_TOKEN... where by user details allow access control

**Pull request merged**